mobile-app/app/screens/AppNavigator/screens/Dex/DexConfirmAddLiquidityV2.tsx

```diff
@@ -0,0 +1,326 @@
+import { RootState } from '@store'
+import { hasTxQueued as hasBroadcastQueued } from '@store/ocean'
+import { hasTxQueued, transactionQueue } from '@store/transaction_queue'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import { DexParamList } from './DexNavigator'
+import { onTransactionBroadcast } from '@api/transaction/transaction_commands'
+import { useAppDispatch } from '@hooks/useAppDispatch'
+import { SummaryTitleV2 } from '@components/SummaryTitleV2'
+import { useWalletContext } from '@shared-contexts/WalletContext'
+import { useAddressLabel } from '@hooks/useAddressLabel'
+import { NativeLoggingProps, useLogger } from '@shared-contexts/NativeLoggingProvider'
+import { NumberRowV2 } from '@components/NumberRowV2'
+import { useTokenPrice } from '../Portfolio/hooks/TokenPrice'
+import { SubmitButtonGroupV2 } from '@components/SubmitButtonGroupV2'
+
+type Props = StackScreenProps<DexParamList, 'ConfirmAddLiquidity'>
+
+export function ConfirmAddLiquidityScreenV2 ({ route }: Props): JSX.Element {
+  const hasPendingJob = useSelector((state: RootState) => hasTxQueued(state.transactionQueue))
+  const hasPendingBroadcastJob = useSelector((state: RootState) => hasBroadcastQueued(state.ocean))
+  const {
+    pair,
+    conversion,
+    pairInfo,
+    summary: {
+      fee,
+      percentage,
+      tokenAAmount,
+      tokenBAmount,
+      lmTotalTokens
+    }
+  } = route.params
+  const dispatch = useAppDispatch()
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const lmTokenAmount = percentage.times(pair.totalLiquidity.token)
+  const [isOnPage, setIsOnPage] = useState<boolean>(true)
+  const navigation = useNavigation<NavigationProp<DexParamList>>()
+  const logger = useLogger()
+  const { address } = useWalletContext()
+  const addressLabel = useAddressLabel(address)
+  const { getTokenPrice } = useTokenPrice()
+
+  useEffect(() => {
+    setIsOnPage(true)
+    return () => {
+      setIsOnPage(false)
+    }
+  }, [])
+
+  async function addLiquidity (): Promise<void> {
+    if (hasPendingJob || hasPendingBroadcastJob) {
+      return
+    }
+    setIsSubmitting(true)
+    await constructSignedAddLiqAndSend(
+      {
+        tokenASymbol: pair.tokenA.displaySymbol,
+        tokenAId: Number(pair.tokenA.id),
+        tokenAAmount,
+        tokenBSymbol: pair.tokenB.displaySymbol,
+        tokenBId: Number(pair.tokenB.id),
+        tokenBAmount,
+        lmTotalTokens: lmTotalTokens
+      },
+      dispatch,
+      () => {
+        onTransactionBroadcast(isOnPage, navigation.dispatch)
+      },
+      logger
+    )
+    setIsSubmitting(false)
+  }
+
+  function onCancel (): void {
+    if (!isSubmitting) {
+      navigation.navigate({
+        name: 'AddLiquidity',
+        params: { pair, pairInfo },
+        merge: true
+      })
+    }
+  }
+
+  return (
+    <ThemedScrollViewV2 style={tailwind('py-8 px-5')}>
+      <ThemedViewV2 style={tailwind('flex-col py-4 mb-4')}>
```

Comment:
```suggestion
      <ThemedViewV2 style={tailwind('flex-col pb-4 mb-4')}>
```
has extra top padding

Reply:
done